WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageParentViewModel.kt

```diff
@@ -128,4 +153,26 @@ class PageParentViewModel
         }
         return children + grandchildren
     }
+
+    internal fun filterListByQuery(list: List<PageItem>, query: String): List<PageItem> {
+        if (query.isEmpty()) {
+            return list
+        } else {
+            val filtered = list.filter { pageItem ->
+                (pageItem.type != PARENT) || // keep the TOP_LEVEL_PARENT and DIVIDER types
```

Comment:
Is there a reason why you always show the Top level page on the filtered result list?

Reply:
I struggled a bit to decide on this one. But my thinking was I want to keep access to 'Top Level' option. I assume the user will not be searching for 'Top Level' but can easily change their mind during the search that they want to use 'Top level'. In that case, they have to cancel search first.

Reply:
I think that once you start searching, it's fine to lose the top level (since normally it's always in the first position). I assume that searching means you just want stuff that matches your query.